sdks/go/pkg/beam/core/util/symtab/symtab.go

```diff
@@ -33,14 +32,13 @@ type SymbolTable struct {
 // New creates a new symbol table based on the debug info
 // read from the specified file.
 func New(filename string) (*SymbolTable, error) {
-	b, err := ioutil.ReadFile(filename)
+	f, err := os.Open(filename)
```

Comment:
<!--thread_id:cc_157274930_t; commit:efbc2a94cba7732675576473452f2cbad9533190; resolved:0-->
<!--section:context-quote-->
> **herohde** wrote:
> +1 to close on error, but not main.

<!--section:body-->
+1 to close on error, and a comment that some of the debug formats require the file handle to stay alive.

Reply:
Done.